### DISCUSSION TITLE
Rendering speed

### DISCUSSION THREAD

**Discussion by ronbere** [vladmandic/sdnext#1493] [Q&A] [answered]
I use sdnext and A1111, both updated. I'm looking at a big difference in rendering speed.
I prefer to use sdnext which I find less austere and especially with regular monitoring in order to improve the server.
Nevertheless, with the same prompt, same negatives. dimension 512x512 with ultrasharp upscaler, the same steps, so the exact copy of the configuration in txt2img, I get this...Vladimir would you have any idea about this?

sdnext
Time taken: 50.21s |

GPU active 8933 MB reserved 11484 MB | System peak 12288 MB total 12288 MB

A1111
Time taken: 21.52s

Torch active/reserved: 6418/6544 MiB, Sys VRAM: 8886/12288 MiB (72.31%)

I'm not an expert at all, I just note that one server gives me active gpu and the other active torch, is this the problem?

**Comment by vladmandic** [comment, marked as answer]
given you're running at nearly-exact half performance, i'd check if youre in fp32 instead fp16 mode or running with no-half.
more than that, i have no idea since you haven't even written what's your platform.

**Reply by ronbere** [reply]
you're right, I was in no-half on sdnext, well seen and thank you
